scylla/src/transport/session.rs

```diff
@@ -625,7 +625,12 @@ impl Session {
         values: impl SerializeRow,
     ) -> Result<QueryResult, QueryError> {
         let query = query.into();
-        self.query_inner(&query, values, None, None).await
+        let (result, paging_state) = self.query_inner(&query, values, None, None).await?;
+        if !paging_state.finished() {
+            warn!("Unpaged query's response hints that there are more pages! Consider using Session API for paging queries,
+                   such as Session::query_iter or Session::query_single_page in a loop.");
+        }
```

Comment:
Here it's the same - this warning means that either driver or Scylla are broken, right? We should imo panic in such situation.